mypy/partially_defined.py

```diff
@@ -396,3 +418,19 @@ def visit_with_stmt(self, o: WithStmt) -> None:
             expr.accept(self)
             self.process_lvalue(idx)
         o.body.accept(self)
+
+    def visit_import(self, o: Import) -> None:
+        for mod, alias in o.ids:
```

Comment:
I think you need to split `id`s with `.` (not 100% sure)

Reply:
Good catch on both of these! Done.